src/stress.codegen/LoadSuiteConfig.cs

```diff
@@ -26,35 +26,14 @@ public LoadSuiteConfig()
 
         public void Serialize(string path)
         {
-            // Serialize the RunConfiguration
-            JsonSerializer serializer = JsonSerializer.CreateDefault();
-
-            using (FileStream fs = new FileStream(path, FileMode.Create))
-            {
-                using (StreamWriter writer = new StreamWriter(fs))
-                {
-                    serializer.Serialize(writer, this);
-                }
-            }
+            File.WriteAllText(path, JsonConvert.SerializeObject(this));
         }
 
         public static LoadSuiteConfig Deserialize(string path)
         {
             LoadSuiteConfig config = null;
 
-            // Deserialize the RunConfiguration
-            JsonSerializer serializer = JsonSerializer.CreateDefault();
-
-            using (FileStream fs = new FileStream(path, FileMode.Open, FileAccess.Read))
-            {
-                using (StreamReader reader = new StreamReader(fs))
-                {
-                    JsonTextReader jReader = new JsonTextReader(reader);
-
-                    // Call the Deserialize method to restore the object's state.
-                    config = serializer.Deserialize<LoadSuiteConfig>(jReader);
-                }
-            }
+            config = JsonConvert.DeserializeObject<LoadSuiteConfig>(File.ReadAllText(path));
```

Comment:
I think we should revert the changes to this file since they're not necessary for the updates your making in the pull request.